scodec/shared/src/main/scala/fs2/scodec/ByteVectorChunk.scala

```diff
@@ -0,0 +1,74 @@
+package fs2
+package scodec
+
+import scala.reflect.{ClassTag, classTag}
+import _root_.scodec.bits.ByteVector
```

Comment:
The package name `fs2.scodec` means root imports may be likely in other code, too.  Open to suggestions.